pfcpiface/main.go

```diff
@@ -52,6 +70,14 @@ type CPIfaceInfo struct {
 	FQDNHost string `json:"hostname"`
 }
 
+// PFCPIfaceInfo : PFCPIface interface settings
+type PFCPIfaceInfo struct {
+	AccessIP    string `json:"access_ip"`
+	P4rtcServer string `json:"p4rtc_server"`
+	P4rtcPort   uint32 `json:"p4rtc_port"`
```

Comment:
can we just keep it string and get rid of conversions back and forth? can remove strconv

Reply:
fixed